content/en/Platform Deep Dive/Collaboration/Organization/Organization Settings/SAML SSO/saml-migration.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "SAML Migration: Update Your Configuration"
+linkTitle: "Update SAML"
+weight: 30
+description: >
+  Learn how to update your SAML configuration from legacy to the new setup.
+---
+
+{{< pageinfo >}}
+<p>On April 24, 2023, we made some changes to our SAML implementation. To avoid any disruptions to authentication, update your SAML configuration.</p><p>During the transition period, your existing SAML SSO setup will be valid. After the transition period, we'll stop supporting legacy configurations.</p>
+{{< /pageinfo >}}
+
+This page is for [Organization Owners](/getting-started/glossary/#organization-owner) who have set up their SAML SSO configuration **before April 24, 2023**.
+
+- If you don't update your SAML configuration, users in your organization will have to authenticate with their email and password or using Google authentication (OAuth 2.0), if relevant.
```

Comment:
if you don't update your SAML configuration by the end of the transition period, we will deprecate your legacy configuration settings. This means you will be unable to sign in to Cobalt. TBD on what we do with the enforcement, we'll probably want to turn off the enforcement, but i don't have clarity on this yet.